Makefile

```diff
@@ -51,7 +51,7 @@ build-cli: clean build-cli-linux build-cli-mac ## Build the CLI
 init-package: ## Create the zarf init package
 	$(ZARF_BIN) package create --confirm
 	mv zarf-init.tar.zst build
-	cd build && sha256sum -b zarf* > zarf.sha256
+	cd build && shasum -a 256 -b zarf* > zarf.sha256
```

Comment:
@jeff-mccoy reason behind this change? The test harness that CI uses doesn't have `shasum` installed, but it does have `sha256sum`. Is there any harm in changing it back to using `sha256sum`?

Reply:
thanks I changed that back and left a note, it was for macos, but there's an easy fix using brew.